### PR TITLE
feat(connector-api): postman collection [CT-1709]

### PR DESCRIPTION
**Pull request by vanjasmailovic**

### CODE REVIEW

File: Connector API.postman_collection.json [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
...mn.\n\nEnter the `client_id` and `client_secret` values from your Console app.\n\nTo get the `report_id` value, you must build a Tink Link URL. To do this, open Console and go to **Income Check** > **Tink Link**, and create your own Tink Link. These steps are required for providing consent to a financial institution, so that Tink can connect and retrieve account data. At the bottom of the page, select **Preview** and follow all steps on the Tink Link page that opens in a new window. At the end of a successful authorization flow, the **Income Check successful** page displays your `income_check_id`. Copy this code, go to Postman, and paste it into the variables field for `report_id`.\n\nMake sure that all of your variables are entered into the **CURRENT VALUE** column. Select **Save** (or Ctrl+S).\n\nWhen you've entered all your variables except for `access_token`, run the **POST** command to exchange your `code` for an `access_token`. The value for `access_token` is automatically entered into your list of variables.\n\nConfirm that your `access_token` has been added to your list of variables. Use your `access_token` any number of times to run the **GET** commands to fetch a report."
```
Comment (jangillich) [inline Connector API.postman_collection.json]: It says "income check" here? Is that intended?

**Review by jangillich — CHANGES_REQUESTED** [review]
Nice! I had a small comment on that it says "income check" in the description in a lot of places, otherwise looks great to me :)

File: Connector API.postman_collection.json (hunk 1 above)
Comment (vanjasmailovic) [reply Connector API.postman_collection.json]: Good one, will fix

**Review by jangillich — APPROVED** [review]
🥇

**Comment by vanjasmailovic** [comment]
bors merge

**Pull request merged**